mysql/mysqldump.go

```diff
@@ -236,10 +277,17 @@ func processConstraint(conv *internal.Conv, table string, constraint *ast.Constr
 
 // toSchemaKeys converts a string list of MySQL primary keys to
```

Comment:
Suggest changing the description of the function rather than adding and "also used for " comment.

How about:

// toSchemaKeys converts a string list of MySQL keys to schema keys.
// Note that we map all MySQL keys to ascending ordered schema keys.
// For primary keys: this is fine because MySQL primary keys are always ascending.
// However, for non-primary keys (aka indexes) this is incorrect: we are dropping
// the MySQL key order specification.
// TODO: Resolve ordering issue for non-primary keys.

Reply:
added function comment